apps/main-app/src/components/paralogy/paralogyFilteredTable.js

```diff
@@ -0,0 +1,235 @@
+import ControlsContainer from '../controlsContainer';
+import { STRINGENCY_HIGH } from '../orthology/constants';
+import {
+  compareAlphabeticalCaseInsensitive,
+  orthologyMeetsStringency
+} from '../../lib/utils';
+import HelpPopup from '../helpPopup';
+import OrthologyFilterHelp from '../orthology/orthologyFilterHelp';
+import useResettableState from '../../hooks/useResettableState';
+import useGeneParalogy from '../../hooks/useGeneParalogy';
+
+const ParalogyFilteredTable = ({geneId}) => {
+  const [
+    filterScoreGreaterThan,
+    setFilterScoreGreaterThan,
+    resetFilterScoreGreaterThan
+  ] = useResettableState(0);
+  const [
+    filterMethod,
+    setFilterMethod,
+    resetFilterMethod
+  ] = useResettableState(null);
+  const [
+    filterBest,
+    setFilterBest,
+    resetFilterBest
+  ] = useResettableState(false);
+  const [
+    filterReverseBest,
+    setFilterReverseBest,
+    resetFilterReverseBest
+  ] = useResettableState(false);
+  const [
+    filterSpecies,
+    setFilterSpecies,
+    resetFilterSpecies
+  ] = useResettableState(null);
+  const [
+    stringencyLevel,
+    setStringencyLevel,
+    resetStringencyLevel
+  ] = useResettableState(STRINGENCY_HIGH);
+  const [showFilterPanel, setShowFilterPanel] = useState(false);
+
+  const { data, isLoading } = useGeneParalogy(geneId);
+
+  const filterCallback = (dat) => {
+    const meetMethodFilter = filterMethod ?
+      dat.predictionMethodsMatched.indexOf(filterMethod) > -1 :
+      true;
+    return (
+      meetMethodFilter &&
+      dat.predictionMethodsMatched.length > filterScoreGreaterThan &&
+      (filterBest ? isBest(dat.best) : true) &&
+      (filterReverseBest ? isBest(dat.bestReverse) : true) &&
+      (filterSpecies ? getOrthologSpeciesName(dat) === filterSpecies : true) &&
+      orthologyMeetsStringency(dat, stringencyLevel)
+    );
+  };
+
+  const resetFilters = () => {
+    resetFilterScoreGreaterThan();
+    resetFilterMethod();
+    resetFilterBest();
+    resetFilterReverseBest();
+    resetFilterSpecies();
+    resetStringencyLevel();
+  };
+
+  if (isLoading) {
+    return <LoadingSpinner />;
+  }
+
+  if (data.total === 0) {
+    return <NoData />;
+  }
+
+  const filteredData = data.results.filter(filterCallback);
+  const all_methods = data.results[0].predictionMethodsMatched.concat(
+    data.results[0].predictionMethodsNotCalled,
+    data.results[0].predictionMethodsNotMatched
+  ).sort(compareAlphabeticalCaseInsensitive);
+
+  const labelStyle = {
+    margin: '0em 1em 0em 0',
+    lineHeight: '2em',
+  };
+  const inputStyle = {
+    margin: '0 0.5em'
+  };
+
+  const buttonStyle = {
+    marginTop: '0.5em',
+    marginRight: '0.5em',
+    minWidth: '8em',
+  };
+
+  const docTextStyle = {
+    fontStyle: 'italic',
+    opacity: 0.7,
+    fontSize: '0.9em',
+  };
+
+  return (
+    <div>
+      <ControlsContainer>
+        <span className='pull-right'>
+          <HelpPopup id='orthology-controls-help'>
```

Comment:
does this need to say 'paralogy-controls-help' or is it exactly the same as the orthology case? Then it would be good to rename it 'homology-controls-help'...

Reply:
It seems to apply the same help, I renamed the id to paralogy-controls-help